data/hai/hai reveal 4 middle.txt

```diff
@@ -1548,10 +1941,48 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			`	"All the Council was silenced in awe as Aimet Taepip spoke in favor of Teeneep, against the secrecy doctrine, presenting a stark choice. Secrecy and safety now conflict; we must choose one or the other: Teeneep's path to safety, or Gemnali's path to secrecy. At this, some brought to attention what traditionalists have said for years: the remarkable similarity between Teeneep's life and that of Eyrisen from the Mythos. The Mythos and Aimet together convinced many Elders to back down, but not enough.`
+			`	"Today's decision... was indecision. We await another path through the chaos that slowly engulfs us. There are rumors of a petition that may reach its threshold in weeks. Perhaps an election will bring us rapidly to our salvation or doom before time runs out."`
+			`	A Hai next to you tells you, "It's all a bit disconcerting, isn't it? Pirates and Unfettered on our borders and Elders doing nothing. A chaos magnet causing mischief while a human mercenary helps her. I wish I was there to see the proceedings, but I know they weren't recorded."`
+			choice
+				`	"Well, you just have to keep living your life, right?"`
+				`	"I'm sure the mercenary will fix all of this."`
+				`	"Political discussions are always droll, regardless of their consequences."`
+			`	As he shrugs and walks away, you realize your acute hunger. Foraging for food in a nearby cafeteria, you soon find Alondo and Remington. They have a surprisingly civilized conversation the entire time. Albeit mostly about work.`
+				goto panic
+
+
+			label return7
+			`	After a minute of walking, you arrive at an empty room. "It seems Teeneep and Sayari have departed to have private discussions, but I can tell you where to meet with the human diplomats. If you prefer, I can take you to a summary of the proceedings instead."`
+			choice
+				`	"Yes, I certainly want the summary."`
+				`	"No, it's time for dinner."`
+					goto spydinner
+			`	"As you wish," she says. After a minute or two listening to her chat about the importance of Teeneep's work, you arrive at a common area with a holoprojector. Before she departs, her face turns dead serious and she whispers, "After this all blows over, you'll be needed up north. Keep your eyes in that direction, Captain; many lives will depend on it."`
+			`	You open your mouth to respond, but she quickly vanishes into the crowd. Surrounding you are Elders, functionaries, and diplomats, all awaiting the news report that is about to begin.`
+				goto news
+			label spydinner
+			`	"As you wish," she says. After a minute or two listening to her chat about the importance of Teeneep's work, you arrive at a cafeteria. She helps you find the best dishes, and leads you to the seating area. Before she departs, her face turns dead serious and she whispers, "After this all blows over, you'll be needed up north. Keep your eyes in that direction, Captain; many lives will depend on it." You open your mouth to respond, but she quickly vanishes into the crowd.`
```

Comment:
And now, with the changes above, if you take this route and only this route, does the hand that holds the knife reveal itself. It does so because, by having skipped the events to nap, you have now become the knife.